playground/scrape-options.js

```diff
@@ -0,0 +1,15 @@
+import moment from 'moment';
+
+// To setup the playground you should do the following:
+// 1. run command 'npm install'.
+// 2. run command 'npm run playground:prepare'.
+// 3. open file '.playground-lib/scrape-options.js` and add your configuration.
+
+// Note: You should modify this file only if it is in folder `.playground-lib`.
+export default {
+  showBrowser: true,
+  scraperId: '', // hapoalim, leumi, discount, leumiCard, visaCal, isracard, amex
+  startDate: moment().add(-2, 'months'),
+  combineInstallments: false,
+  saveLocation: `${__dirname}/output`,
+};
```

Comment:
@esakal as mentioned above, I would not want this to be setup under the repo.
My favorite option would be to supply these as command line parameters (we could provide sample `bash` scripts).
The other option would be to instruct the user to save these settings under the config folder.

Reply:
> My favorite option would be to supply these as command line parameters

It is possible, yet I don't see the big advantage of having them as a cli arguments unless you want this feature to be added to ynab-updater as an alternative to the existing flow which requires user interactions

> The other option would be to instruct the user to save these settings under the config folder

I personally try to avoid forcing the user to perform manual step if possible as I find them error prone and confusing.

Reply:
@esakal ok.
But can we save it for the user under `${os.homedir()}/.bank-scrapers-playground`?